grub-core/loader/i386/slaunch.c

```diff
@@ -91,20 +91,20 @@ grub_cmd_slaunch (grub_command_t cmd __attribute__ ((unused)),
     return grub_error (GRUB_ERR_BAD_ARGUMENT, N_("argument expected"));
 
   /* Should be executing on the BSP  */
-  grub_printf("%s:%d: about to do rdmsr: ", __FUNCTION__, __LINE__);
+  grub_dprintf ("slaunch", "about to do rdmsr: ");
```

Comment:
@miczyg1 It seems that output is buffered and flushed on new line, so if `grub_rdmsr()` fails, this line will not be printed. Can you add `\r\n` here, and perhaps indent the following "OK"? Or just drop those lines altogether, these really shouldn't fail now and do not provide any useful information anymore.

Reply:
Removed these